IndicoToolkit.Tests/FileProcessingTest.cs

```diff
@@ -0,0 +1,75 @@
+using Xunit;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+
+namespace IndicoToolkit.Tests
+{
+    public class FileProcessingTest
+    {
+        [Fact]
+        public void TestGetFilePathsFromDirNull()
+        {

```

Comment:
isn't this test just that filecount is 0 when instantiated? i.e. what does FromDirNull mean before you run anything? (I was thinking the test would be pointing at an empty directory should yield empty count)